### ISSUE TITLE
Better accessibility

### ISSUE THREAD

**Issue by svanlaere** [typemill/typemill#324]
Typemill needs improvements on its accessibility, can we collaborate on this?

**Comment by trendschau** [comment]
Yes, highly welcome! I don't have a lot of time right now but let us try to improve it step by step.
